src/coreclr/inc/daccess.h

```diff
@@ -1104,7 +1104,7 @@ class __DPtrBase : public __TPtrBase
     {
         return DPtrType(DacTAddrOffset(m_addr, val, sizeof(type)));
     }
-#if defined (HOST_64BIT)
+#if defined (HOST_64BIT) || defined(HOST_WASM)
```

Comment:
Where is this needed when WASM is a 32 bit platform and no other 32 bit platform needs that?

Reply:
Interestingly this doesn't fail anymore. Either it was hit in a code in disabled feature or it changed in main meanwhile. I will remove it.